connectors/cli/index.py

```diff
@@ -36,6 +36,9 @@ async def __list_indices(self):
         try:
             return await self.cli_client.list_indices()
         except ApiError as e:
+            # If the API is not available, we are in serverless mode
+            if e.error == "api_not_available_exception":
+                return await self.cli_client.list_indices_serverless()
```

Comment:
Should we log something about it?

Reply:
I couldn't find a good reason why we should log this.

Reply:
Mainly as a way to explain why we cannot show number of documents in the indices